docs/source/developer/baf_minikube_setup.md

```diff
@@ -73,36 +88,75 @@
 1. Edit the following configurations in the `network.yaml` correctly:
    ```yaml
    docker:
-      url: "index.docker.io/hyperledgerlabs"
-      username: “<your docker username>"
-      password: "<your docker password/token>"
+     url: "index.docker.io/hyperledgerlabs"
+     username: “<your docker username>"
+     password: "<your docker password/token>"
    ```
 1. For each `organization`, update the following: (there are 3 organizations in this sample network.yaml)
+
    ```yaml
    vault:
-      url: http://<Your Vault server address>:8200 (Use the local IP address rather than localhost)
-      root_token: <your vault_root_token>
+     url: http://<Your Vault server address>:8200 (Use the local IP address rather than localhost)
+     root_token: <your vault_root_token>
    gitops:
-      git_ssh: <ssh url of your forked repo>
-      git_push_url: <https url of your forked repo without the https://)
-      username: "<github_username>"          
-      password: "<github_token>"         
-      email: "<github_email>"
+     git_ssh: <ssh url of your forked repo>
+     branch: "develop"
+     git_push_url: <https url of your forked repo without the https://)
+     username: "<github_username>"
+     password: "<github token/password>"
+     email: "<github_email>"
+     private_key: "/home/blockchain-automation-framework/build/gitops.pem"
```

Comment:
"gitops" only, should be hardcoded in the network.yaml sample hence not required here.